merge-opencog-to-singnet.py

```diff
@@ -69,9 +69,12 @@ def run(cmd, stdout=subprocess.DEVNULL, **kargs):
     return subprocess.run(cmd, stderr=subprocess.STDOUT,
                           stdout=stdout, **kargs)
 
-def get_forks(api):
+def get_forks(api, repo_names=[]):
     print("looking for opencog forks")
     singnet_forks = api.get_repos("singnet", type="forks")
+    if repo_names:
+        singnet_forks = [repo for repo in
+                         singnet_forks if repo["name"] in repo_names]
     fork_and_parent = [(repo, api.get_url(repo["url"])["parent"]) for repo in
                    singnet_forks]
```

Comment:
BTW, I noticed a strange indentation here, although it doesn't hurt. I didn't follow it cause I could make sense of that convention, let me know if you have a rational behind it.

Reply:
No problem, I am not a Python guru and usually just use automatic indentation by vim plugin.